docs/.vuepress/theme/components/events/Events.vue

```diff
@@ -25,7 +25,7 @@
         <div class="upcoming-event-list">
           
           <div class="event-card">
-            <img src="./LiveDemo_Steph.webp" alt="Live Demo" />
+            <img src="./meshStack_YT_LiveDemo_Mar_2024.webp" alt="Live Demo" />
```

Comment:
```suggestion
            <img src="./meshStack_YT_LiveDemo_Mar_2024.webp" alt="Live Demo" style="border-radius: 45px;"/>
```

I think Polina would also look better with some curves ✨